server/utils/shabads/is-list-of-shabads.js

```diff
@@ -0,0 +1,3 @@
+export const isListOfShabads = object => {
```

Comment:
Somehow, I can't see the need of having this here, since you are checking property that is hardcoded here which may stop others to re-use it.

**Suggestion:**
I case you really need a function then either pass property name as param or keep this function inside create-metadata-from-response.js file only.

Happy to discuss further.